msal-python-conceptual/advanced/managed-identity.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Using Managed Identity
+description: Learn how to use Managed Identity with Microsoft Authentication Library (MSAL) for Python.
+author: localden
+
+ms.service: msal
+ms.subservice: msal-python
+ms.topic: conceptual
+ms.date: 06/25/2024
+ms.author: ddelimarsky
+ms.reviewer: rayluo
+---
+
+# Using Managed Identity
+
+[Managed identity](/entra/identity/managed-identities-azure-resources/overview) enables developers to remove the need to store, rotate, and otherwise handle credentials for their applications running in the Microsoft Azure cloud. Microsoft Authentication Library (MSAL) for Python supports using authentication with managed identity on supported Azure workloads.
+
+>[!NOTE]
+>Managed Identity in MSAL Python is supported since version [1.29.0](https://pypi.org/project/msal/1.29.0/) of the [`msal` package](https://pypi.org/project/msal/).
+
+MSAL Python supports acquiring tokens through the managed identity service when used with applications running inside Azure infrastructure, such as:
+
+- [Azure App Service](https://azure.microsoft.com/products/app-service/) (API version `2019-08-01` and above)
+- [Azure VMs](https://azure.microsoft.com/free/virtual-machines/)
+- [Azure Arc](/azure/azure-arc/overview)
+- [Azure Cloud Shell](/azure/cloud-shell/overview)
+- [Azure Service Fabric](/azure/service-fabric/service-fabric-overview)
```

Comment:
Python also supports Azure ML https://learn.microsoft.com/en-us/azure/machine-learning/how-to-identity-based-service-authentication?view=azureml-api-2&tabs=cli